### PR TITLE
feat: Add link to blog on yearn.finance in footer 

### PR DESCRIPTION
**Pull request by nuggetnchill**
Issue #564 

Just add Blog link to the footer, pretty straight forward.

### CODE REVIEW

**Comment by turtlemoji** [comment]
We need to check if it is a desired icon, also we refactored the footer on another branch @nuggetnchill, we should merge it soon, I will ping you here when it is merged!

**Comment by FoxTheSin** [comment]
order should be:
disclaim docu blog governan securi

please and thank you!

**Comment by nuggetnchill** [comment]
resolved conflicts of the new merge, and fix the order 😄

File: src/client/components/app/Footer/index.tsx [hunk 1]
```diff
@@ -21,6 +21,18 @@ const footerLinks = [
     link: 'https://gov.yearn.finance',
     name: 'gov',
   },
+  {
+    link: 'https://blog.yearn.finance/',
+    name: 'blog',
+  },
+  // {
```
Comment (turtlemoji) [inline src/client/components/app/Footer/index.tsx:28]: remove comments

File: public/locales/en/common.json [hunk 2]
```diff
@@ -122,8 +122,9 @@
   "footer": {
     "links": {
       "disclaimer": "Disclaimer",
-      "docs": "Documentation",
-      "gov": "Governance forum",
+      "docs": "Docs",
```
Comment (turtlemoji) [inline public/locales/en/common.json:125]: image
These are the texts in the current design, are these changes approved by copy/design team? @FoxTheSin

Comment (turtlemoji) [reply public/locales/en/common.json:125]: Approved by Evan after fox asking, resolved

**Review by FoxTheSin — DISMISSED** [review]

File: src/client/components/app/Footer/index.tsx [hunk 3]
```diff
@@ -21,6 +21,18 @@ const footerLinks = [
...
+  //   link: 'https://v1.yearn.finance',
+  //   name: 'v1',
+  // },
+  // {
+  //   link: 'https://yearn.finance',
+  //   name: 'v2',
+  // },
```
Comment (turtlemoji) [inline src/client/components/app/Footer/index.tsx:35]:
```suggestion
```

File: public/locales/en/common.json [hunk 4]
```diff
@@ -122,8 +122,9 @@
...
+      "blog": "Blog",
+      "gov": "Gov",
```
Comment (turtlemoji) [inline public/locales/en/common.json:127]:
```suggestion
      "gov": "Governance",
```

File: public/locales/en/common.json (hunk 2 above)
Comment (turtlemoji) [inline public/locales/en/common.json:125]:
```suggestion
      "docs": "Documentation",
```

**Review by turtlemoji — APPROVED** [review]

**Review by FoxTheSin — APPROVED** [review]

**Pull request merged**